spyder/plugins/findinfiles/widgets/main_widget.py

```diff
@@ -630,7 +635,7 @@ def set_max_results(self, value=None):
             dialog.setWindowTitle(_('Max results'))
             dialog.setLabelText(_('Set maximum number of results: '))
             dialog.setInputMode(QInputDialog.IntInput)
-            dialog.setIntRange(1, 10000)
+            dialog.setIntRange(5, 10000)
```

Comment:
Just in case, this means that now a search can not be limited to one result, right @ccordoba12 ?

Reply:
Yeah, that's correct. The fix I did works if the minimal number of max results that users can set is 2. Since that number seems a bit arbitrary, I decided to increase it to 5.

Reply:
Maybe we should add here a comment indicating the need for this to be greater than 2 in order to prevent issues and referencing the related issue?

Reply:
Good idea, will do.